### PR TITLE
code normalization and fix PSR-2

### PR DESCRIPTION
**Pull request by lgcosta**

### CODE REVIEW

File: httploader/index.php [hunk 1]
```diff
@@ -69,16 +69,16 @@
...
+        <link href="../../Themes/Centreon-2/style.css" rel="stylesheet" type="text/css"/>
+        <link href="../../Themes/Centreon-2/jquery-ui/jquery-ui.css" rel="stylesheet" type="text/css"/>
+        <link href="../../Themes/Centreon-2/jquery-ui/jquery-ui-centreon.css" rel="stylesheet" type="text/css"/>
+        <script type="text/javascript" src="../../include/common/javascript/jquery/jquery.js"></script>
+        <script type="text/javascript" src="../../include/common/javascript/jquery/jquery-ui.js"></script>
         <script type="text/javascript" src="../../include/common/javascript/widgetUtils.js"></script>
     </head>
     <body>
-	<iframe id="test" width="100%" height="900px"></iframe>
+    <iframe id="test" width="100%" height="900px"></iframe>
```
Comment (sc979) [inline httploader/index.php]: wrong indentation

File: httploader/index.php [hunk 2]
```diff
@@ -69,16 +69,16 @@
...
     </body>
 <script type="text/javascript">
 var widgetId = <?php echo $widgetId; ?>;
```
Comment (sc979) [inline httploader/index.php]: wrong indentation

Comment (lgcosta) [reply httploader/index.php]: done!

File: httploader/index.php (hunk 1 above)
Comment (lgcosta) [reply httploader/index.php]: done!

**Pull request merged**